go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -614,6 +614,46 @@ func getOVNSBZone() (string, error) {
 	return dbZone, nil
 }
 
+/** HACK BEGIN **/
+// TODO(tssurya): Remove this HACK a few months from now.
+// checkOVNSBNodeLRSR returns true if the logical router static route for the
+// the given nodeSubnet is present in the SBDB
+func checkOVNSBNodeLRSR(nodeSubnet string) bool {
+	match := fmt.Sprintf("match=\"reg7 == 0 && ip4.dst == %s\"", nodeSubnet)
```

Comment:
the match doesnt work for ipv6 cause i think you need to escape the :

Reply:
oopsy i need to fix this.

Reply:
yeah the problem was `ip4.dst`, had to use `ip6.dst` for this to work. its fixed.